package.json

```diff
@@ -67,34 +65,34 @@
     "husky": "0.14.3",
     "ignore-styles": "5.0.1",
     "mem-storage-area": "1.0.3",
-    "mocha": "4.0.1",
+    "mocha": "5.0.1",
     "mocha-jenkins-reporter": "0.3.10",
-    "nodemon": "1.14.11",
+    "nodemon": "1.15.1",
     "npm-run-all": "4.1.2",
     "nyc": "11.4.1",
     "shx": "0.2.2",
-    "sinon": "4.1.2",
-    "sinon-chrome": "2.2.1",
-    "standard": "10.0.3",
+    "sinon": "4.3.0",
+    "sinon-chrome": "2.2.4",
+    "standard": "11.0.0",
     "uglifyify": "4.0.5",
-    "watchify": "3.9.0",
-    "web-ext": "2.3.2"
+    "watchify": "3.10.0",
+    "web-ext": "2.4.0"
   },
   "dependencies": {
-    "choo": "6.6.1",
+    "choo": "6.8.0",
     "doc-sniff": "1.0.1",
-    "file-type": "7.3.0",
-    "ipfs": "0.27.6",
-    "ipfs-api": "17.2.7",
-    "ipfs-css": "0.1.0",
-    "ipfs-postmsg-proxy": "2.7.0",
+    "file-type": "7.6.0",
+    "ipfs": "0.27.7",
+    "ipfs-api": "18.1.1",
+    "ipfs-css": "0.2.0",
+    "ipfs-postmsg-proxy": "2.7.1",
     "is-ipfs": "0.3.2",
-    "is-svg": "2.1.0",
+    "is-svg": "3.0.0",
     "lru_map": "0.3.3",
-    "mime-types": "2.1.17",
+    "mime-types": "2.1.18",
     "p-queue": "2.3.0",
     "prundupify": "1.0.0",
-    "tachyons": "4.9.0",
-    "webextension-polyfill": "0.1.2"
+    "tachyons": "4.9.1",
+    "webextension-polyfill": "0.2.1"
```

Comment:
@alanshaw this one needs to stay in `0.1.2` due to known regression in Chrome: https://github.com/mozilla/webextension-polyfill/issues/70 :cactus: